submissions/Nemooochka/friends-app/js/filters.js

```diff
@@ -0,0 +1,111 @@
+import {Url} from "./url.js";
+
+export class Filters {
+    constructor(users, node) {
+        this.users = users;
+        this.node = node;
+        this.filterByGenderText = 'filterByGender';
+    }
+
+    getCardsChildNodesArr() {
+        let cardsChildNodes = this.node.childNodes;
+        return [...cardsChildNodes];
+    }
+
+    activateFilterByGenderIcon(filterValue) {
+        document.querySelector(`[filter-gender='${filterValue}']`).checked = true;
```

Comment:
please, store this element in some global const in the constructor of this class